code-collab-app/src/homepage.js

```diff
@@ -108,7 +108,6 @@ class Homepage extends React.Component {
             {this.state.showNamePopup ?
               <NamePopup peer={this.props.peer_data} history={this.props.history} closePopup={this.toggleNamePopup.bind(this)}/>
               : null}
-            }
```

Comment:
Is this supposed to be removed??

Reply:
Yeah, this is an extra bracket so it shows up in the top left corner of the homepage 🙂